### ISSUE TITLE
Can't connect to MySQL 'The alien function "SSL_load_error_strings" is undefined.'

### ISSUE THREAD

**Issue by AzRu** [dimitri/pgloader#197]
Pgloader 3.2 on mac osx 10.10.2.

This is what's in the command file:
```
 LOAD DATABASE
 from      mysql://user:pass@<instance-name>.us-east-1.rds.amazonaws.com:3306/dbname
 into postgresql://user:pass@<instance-name>.us-east-1.rds.amazonaws.com:5432/dbname

 WITH include drop, create tables, no truncate,  
      create indexes, reset sequences, foreign keys  
 
 SET maintenance_work_mem to '128MB', work_mem to '12MB', search_path to 'dbname'  
 
 CAST type datetime to timestamptz  
                drop default drop not null using zero-dates-to-null,  
      type date drop not null drop default using zero-dates-to-null   
  
 BEFORE LOAD DO  
 $$ create schema if not exists dbname; $$;
```

This is the output I get:
```
An unhandled error condition has been signalled:
   Failed to connect to mysql at "<instance-name>us-east-1.rds.amazonaws.com" (port 3306) as user "user": The alien function "SSL_load_error_strings" is undefined.
```

**Comment by dimitri** [comment]
Do you have `openssl` installed? I seem to have it installed via `brew` here, it might be possible to use other methods.

**Comment by AzRu** [comment]
I did some digging in the meantime and the issue was that despite brew installing 1.0.2a, OS X comes with 0.9.8 and it was trying that one since I didn't add it to the PATH.

But now I get "The alien function "CRYPTO_num_locks" is undefined."

**Comment by dimitri** [comment]
See https://github.com/dimitri/pgloader/issues/149#issuecomment-75272361 ; can you test compiling pgloader from git?

**Comment by AzRu** [comment]
Thanks for the help but I just dumped the data on the RDS instance to a local db and it worked fine that way. Unfortunately I don't have time to debug the issue so I'm gonna close this.

**Comment by dimitri** [comment]
Good practical move. Thanks for following up the bug report. Should be fixed in 3.2.1, will work on releasing that.

**Comment by AzRu** [comment]
Awesome, I work at a startup so things  go so fast and I can't waste time.
Thanks a lot for this tool by the way. The one I tried previously didn't
handle foreign keys. Can you believe that?
